### PR TITLE
Add Initial OSR SIG 2022 Roadmap

### PR DESCRIPTION
**Pull request by mcleo-d**
## Description
This pull request adds an initial OSR SIG roadmap to the `osr` microsite taken from GitHub Discussion https://github.com/finos/open-source-readiness/discussions/23.

The following headers describe the `roadmap.md` page added to the `osr` repo and the navigational changes to the `osr` microsite.

## Additional Markdown Page
`docs/roadmap.md`

## Additional Microsite Navigation
```
{
  "docs": {
    "General": [
      "roadmap",
      "meetings",
      "resources",
      "about"
    ]
  }
}
```

## Additional Microsite Header
```
headerLinks: [
    {doc: 'roadmap', label: 'OSR 2022 ROADMAP'},
    {doc: 'meetings', label: 'MEETINGS'},
    {doc: 'resources', label: 'RESOURCES'},
    {doc: 'about', label: 'ABOUT'},
```

### CODE REVIEW

**Review by psmulovics — APPROVED** [review]

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+To help accelerate the OSR SIG roadmap, the following labels have been added to the OSR repo in GitHub that can be applied to GitHub Issues and Pull Requests.
+
+### Open Source Readiness GitHub Labels
+
+- `osr-relationship-building`
+- `open-source-readiness-promotion`
+- `open-source-maturity-model`
+- `reglatory-requirement`
+- `finacial-institution-challenge`
```
Comment (psmulovics) [inline docs/roadmap.md:55]: Typo, should be 'financial'

File: docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# Implementing the OSR SIG Roadmap
+
+To help accelerate the OSR SIG roadmap, the following labels have been added to the OSR repo in GitHub that can be applied to GitHub Issues and Pull Requests.
+
+### Open Source Readiness GitHub Labels
+
+- `osr-relationship-building`
+- `open-source-readiness-promotion`
+- `open-source-maturity-model`
+- `reglatory-requirement`
```
Comment (psmulovics) [inline docs/roadmap.md:54]: Typo, should be 'regulatory'

**Comment by psmulovics** [comment]
Should not the roadmap.md file named roadmap-2022.md ?

File: docs/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+### Explore and develop regulators role in the Open Source maturity of the industry, in conjunction with the FINOS Innovation Regulation SIG
+
+- Between [this discussion](https://groups.google.com/a/finos.org/g/osr/c/GMMD38RkaMY/m/nYMwTdR3BAAJ) on the list and this [github discussion](https://github.com/finos/open-source-readiness/discussions/9), consolidating a shared list of common regulatory requirements industry is concerned about
+- Then, mapping to existing solutions (or start open sources project to address them).
+
+### Support the definition and evolutionj of a truly industry-wide OSMM
+
+- Provide / solicit ongoing Feedback and validation of OSMM ahead of FINOS OSMM launch
+- Participate and amply FINOS OSMM launch
```
Comment (troups) [inline docs/roadmap.md:33]: amply to amplify? apply, i think its amplify?

File: docs/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- Promote participants and adjacent communities to take the survey
+- Propose training / certifications to strengthen the value proposition of OSMM for all constituents
```
Comment (troups) [inline docs/roadmap.md:35]: @mcleo-d  - do you know of existing training / certification best practice, either in FinOS or LF? Would be interested to see what we're shooting for here.

**Comment by psmulovics** [comment]
I share the sentiment mentioned during the call too of not really feeling this roadmap-y, more like manifesto, goal, etc kind of document - which is fine, we need that too; but when comes to roadmap I more think of something like:

image

**Comment by robmoffat** [comment]
Here is my starter-for-10 as to what the roadmap might contain.  Obviously, up for discussion.

1.  **Interviews with OSPOs**:   Period where we discuss OSR with OSPOs at various financial institutions in order to gauge their concerns and establish what practices they each employ.  
2. **Interviews with Project Owners in Financial Institutions**:  We should elicit feedback from the project owners to establish what the main problems they face are, and what the best practices are
3. **Enhance the Microsite**: With anonymised summaries of the above - best practices, key challenges, process know-how etc.
4.  **Feedback**: perhaps not a milestone, should happen as we go, but getting the contributors involved in reviewing what has been produced
5. **Questionnaire**:  Building an FinTech-specific Open-Source readiness questionnaire based on all the above
6. **Publicise**
7. **Provide Accreditation**
8. **Publish Training Materials**

... just off the top of my head.  Interested to see what everyone else thinks on this

**Comment by mcleo-d** [comment]
@psmulovics and @robmoffat 

Thanks so much for your roadmap feedback in this PR. In order to tighten the scope of the PR deliverable, I have moved your suggestions to the following GitHub Discussion thread so ongoing feedback from the SIG 👍🏻 

https://github.com/finos/open-source-readiness/discussions/23#discussioncomment-2516808

Keep the ideas and suggestions coming!

James.

File: docs/roadmap.md (hunk 1 above)
Comment (mcleo-d) [reply docs/roadmap.md]: Fixed here .. 92570273d8307adfa79f0869d264e2af57cf749f

**Comment by mcleo-d** [comment]
> Should not the roadmap.md file named roadmap-2022.md ?

Fixed here ... 64518f972e65c0e0b5fa33d6ecb3df8b803cdefa

File: docs/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- Through prioritization of a backlog (e.g. github issues) at OSR SIG meetings.
+- Processing output of the [OSMM](https://github.com/finos/osmm) survey responses FINOS receives.
+- Through real-life findings FINOS can feed in (e.g. Member XX cannot use DCOs, Member YY can’t access github, etc).
+
+### Explore and develop regulators role in the Open Source maturity of the industry, in conjunction with the FINOS Innovation Regulation SIG
+
+- Between [this discussion](https://groups.google.com/a/finos.org/g/osr/c/GMMD38RkaMY/m/nYMwTdR3BAAJ) on the list and this [github discussion](https://github.com/finos/open-source-readiness/discussions/9), consolidating a shared list of common regulatory requirements industry is concerned about
+- Then, mapping to existing solutions (or start open sources project to address them).
+
+### Support the definition and evolutionj of a truly industry-wide OSMM
```
[the remark this answers is not in the document]
Comment (mcleo-d) [reply docs/roadmap.md]: Fixed here ... c17a1d837c1256c00af05d66e5143852b799463d

File: docs/roadmap.md (hunk 3 above)
Comment (mcleo-d) [reply docs/roadmap.md]: Fixed here ... 52e4e3f03a160a4969ebe3e082d6e08c6be3cba9

**Comment by mcleo-d** [comment]
@psmulovics, @vmbrasseur and @troups,

I have fixed the direct feedback relating to commits in this PR that are now `ready-for-review`.

It would be awesome if you could accept the changes and charge ahead with the merge! 👍🏻 

James.

**Comment by mcleo-d** [comment]
@troups - I will find a mechanism outside this PR to respond to - https://github.com/finos/open-source-readiness/pull/29#discussion_r844093736

Thanks for your questions 😁 

James.

File: docs/roadmap.md (hunk 4 above)
Comment (mcleo-d) [reply docs/roadmap.md:35]: Hi @troups - Please note, I have referenced this comment here https://github.com/finos/open-source-readiness/pull/27#issuecomment-1094865935 as I would like to start using the `trainings` section of the OSR microsite to help answer this question 👍🏻

**Comment by psmulovics** [comment]
> > Should not the roadmap.md file named roadmap-2022.md ?
> 
> Fixed here ... [64518f9](https://github.com/finos/open-source-readiness/commit/64518f972e65c0e0b5fa33d6ecb3df8b803cdefa)

So that should mean that the sidebar would contain the link to that and the old file would be removed from this PR?

**Pull request closed without merging**